pkg/services/sqlstore/annotation.go

```diff
@@ -142,6 +143,7 @@ func (r *SQLAnnotationRepo) Find(query *annotations.ItemQuery) ([]*annotations.I
 		FROM annotation
 		LEFT OUTER JOIN ` + dialect.Quote("user") + ` as usr on usr.id = annotation.user_id
 		LEFT OUTER JOIN alert on alert.id = annotation.alert_id
+		LEFT OUTER JOIN dashboard on dashboard.id = annotation.dashboard_id
```

Comment:
Since we're introducing another table in the join, there's a small chance of query performance impact here.  I think that's unlikely, since the `dashboard.id` field should (?) be indexed.

Reply:
Thanks for your feedback! dashboard id is the primary key, so it should be indexed :)

Reply:
Good stuff, thanks :)  Reading back I might have added as much noise as useful commentary.. but I hope some of it has been useful.